google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, global_configs):
             api_name=self.API_NAME)
 
         self.rate_limiter = RateLimiter(
-            self.global_configs.get('max_admin_api_calls_per_day'),
+            self.global_configs.get('max_admin_api_calls_per_100_seconds'),
```

Comment:
I've sent the doc out to the team, but yes, essentially it's fine to put the checks in the code for now. The code should handle both config directives and log a warning if a directive will be deprecated in the next version possibly. Let's not try to complicate things for now (ie. don't make a framework for handling deprecation and stuff...) and try to work through a few versions before we actually decide if we need to do more engineering around config management.